src/graph/ExecutionEngine.cpp

```diff
@@ -25,9 +25,10 @@ ExecutionEngine::~ExecutionEngine() {
 
 Status ExecutionEngine::init(std::shared_ptr<folly::IOThreadPoolExecutor> ioExecutor) {
     auto addrs = network::NetworkUtils::toHosts(FLAGS_meta_server_addrs);
-    if (!addrs.ok()) {
+    if (!addrs.ok() || addrs.value().empty()) {
```

Comment:
If FLAGS_meta_server_addrs is not empty,  in which case "toHosts" return empty array?